examples/assets-and-custom-transfer-logic/assets/clawback-escrow.py

```diff
@@ -0,0 +1,82 @@
+# Escrow Clawback Example in pyTeal
+
+# Add parent directory to path so that algobpy can be imported
+import sys
+sys.path.insert(0,'..')
+
+from algobpy.parse import parseArgs
+from pyteal import *
+
+
+def clawback_escrow(ASSET_ID, APP_ID):
+
+	# check properties of txGroup passed
```

Comment:
is indentation correct here ?

Reply:
yes (on my system)
![image](https://user-images.githubusercontent.com/33264364/107943847-49a03f80-6fb3-11eb-85a8-42615a3548a8.png)

Reply:
![ind](https://user-images.githubusercontent.com/26249903/108065338-ddd9d780-7083-11eb-9e28-1b3e1eb0037f.png)

it is good in file though.